tasks/audit-cat2.yml

```diff
@@ -77,3 +77,88 @@
       - RHEL-07-040640
       - always
       - ssh
+
+- name: "MEDIUM | RHEL-07-020840 | PATCH | All local initialization files for interactive users must be owned by the home directory user or root."
+  rhel7_020840: check=True
+  failed_when: no
+  changed_when: no
+  ignore_errors: yes
+  register: rhel_07_020840_audit
+  tags:
+      - cat2
+      - medium
+      - patch
```

Comment:
The module handles the bulk of the load on this instead of a multi-step or extra long command.  Here is the logic of the init_fix.py module

1.  find all users that have the ability to login
2. Get the home directories for these users
3. Pull all files/dirs that start with .
4. Operate on only files
5.  Call the appropriate function fix(user, group, permissions), validate paths.  When operating in check/audit mode it will return any files that do not pass, in fix mode it will resolve any issues that it can.